common/src/foundation/python-packages/packages.json

```diff
@@ -15,23 +15,24 @@
 
 {
 	"Django":		{ "fe": true, "version": "1.11.20" },
-	"Flask":		{ "fe": true }, 
-	"GitPython":            { "fe": true, "bootstrap": true },
-	"PyMySQL":		{ "fe": true, "bootstrap": true, "version": "0.8.1" }, 
+	"Flask":		{ "fe": true },
+	"GitPython":		{ "fe": true, "bootstrap": true },
+	"PyMySQL":		{ "fe": true, "bootstrap": true, "version": "0.8.1" },
 	"ansible":		{ "fe": true },
-	"configparser":		{ "fe": true, "be": true }, 
-	"flake8":		{ "fe": true, "bootstrap": true }, 
+	"configparser":		{ "fe": true, "be": true },
+	"flake8":		{ "fe": true, "bootstrap": true },
 	"jmespath":		{ "fe": true },
 	"jsoncomment":		{ "fe": true, "bootstrap": true },
 	"mod-wsgi":		{ "fe": true },
 	"netifaces":		{ "fe": true, "be":"true", "bootstrap": true },
 	"paramiko":		{ "fe": true },
 	"pexpect":		{ "fe": true },
 	"psutil":		{ "fe": true, "be": true, "bootstrap": true },
-	"python-daemon":	{ "fe": true, "be": true, "bootstrap": true }, 
+	"python-daemon":	{ "fe": true, "be": true, "bootstrap": true },
 	"pyzmq":		{ "fe": true, "be": true, "bootstrap": true, "version": "17.1.2" },
-	"redis":		{ "fe": true, "bootstrap": true }, 
+	"redis":		{ "fe": true, "bootstrap": true },
 	"requests":		{ "fe": true },
 	"testinfra":		{ "fe": true, "bootstrap": true, "version": "1.16.0" },
-	"yoyo-migrations":	{ "fe": true }
+	"yoyo-migrations":	{ "fe": true },
+	"libvirt-python":	{ "fe": true }
```

Comment:
Not to sound like the Ariadne devs... but it would be nice if you added `libvirt-python` after `jsoncomment` so the list stays alphabetized